### PR TITLE
CLOUD-2381 move artifact definitions into consuming modules

### PR DESCRIPTION
**Pull request by rcernich**
Signed-off-by: rcernich <rcernich@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by jmtd — APPROVED** [review]

**Comment by rcernich** [comment]
Hey @ryanemerson, thanks for merging this, but this change will break the image build without the corresponding changes in cct_module.  If you look at the issue in JIRA, you'll see a number of PRs associated with this change.  QE will also sign-off on the changes after testing the changes in isolation (i.e. off of dev builds).  The cct_module changes won't be merged until QE approves the changes as part of the next sprint.

**Comment by ryanemerson** [comment]
@rcernich It seems I was a little hasty in hitting the merge button. Do you think it's necessary to revert the commit? Or only go down that path if we need a 7.1 respin before QE signs off?

\\cc @slaskawi @osnipso for awareness

**Comment by rcernich** [comment]
Let's just wait.  No need to mess up the history if we don't need to.  We will need to be cognizant of this when we merge into the datagrid71 and datagrid71-cp branches after this is released (i.e. we should merge the tree before this commit).

**Pull request merged**
